src/ontology/uberon-edit.obo

```diff
@@ -77373,6 +77373,7 @@ def: "The metacarpophalangeal joints (MCP) are of the condyloid kind, formed by
 subset: pheno_slim
 synonym: "articulationes metacarpophalangeae" RELATED LATIN [Wikipedia:Metacarpophalangeal_joint]
 synonym: "carpometacarpophalangeal joint" NARROW SENSU [NCBITaxon:8782]
+synonym: "knuckles" EXACT [https://orcid.org/0000-0001-5208-3432]
```

Comment:
should be singular.

You can also add plurals, but add a PLURAL annotation

See https://github.com/obophenotype/uberon/wiki/Using-uberon-for-text-mining

(we should turn the wiki into proper docs, and the guidelines here are broader than TM and should be read by all editors)

Reply:
done!